Source/gmenu.cpp

```diff
@@ -349,49 +349,66 @@ BOOLEAN gmenu_valid_mouse_pos(int *plOffset)
 	return 1;
 }
 
-int gmenu_left_mouse(int a1)
+BOOL gmenu_left_mouse(BOOL isDown)
 {
-	int result;      // eax
-	unsigned int v2; // eax
-	unsigned int v3; // eax
-	TMenuItem *v4;   // esi
-	unsigned int v5; // eax
-	//LPARAM v6; // ecx
-	int a1a; // [esp+4h] [ebp-4h]
-
-	if (a1) {
-		if (!sgpCurrentMenu || MouseY >= 352)
-			return 0;
+	BOOL result;
+	TMenuItem *v3;
+	DWORD v4, v5;
+	int dummy;
+
+	if (!isDown) {
+		if (mouseNavigation) {
+			mouseNavigation = 0;
+			result = TRUE;
+		} else {
+			result = FALSE;

```

Comment:
Seems like I had old comment unsubmitted. Sorry to flood the closed PR.

Reply:
yeah, np :P yep, I can't do returns because it has to jump to that only return so I have to store the result in a variable then jump there, clunky as hell but that's how I got it bin exact :)